src/utils/configured-graphql-client.ts

```diff
@@ -0,0 +1,21 @@
+import {GraphQLClient} from 'graphql-request'
+import config from '../lib/config'
+
+export const setAuthorizationHeaderForClient = (
+  client: GraphQLClient,
+  authToken: string | undefined,
+) => {
+  if (authToken) {
+    const authorizationHeader = `Bearer ${authToken}`
+    client.setHeader('authorization', authorizationHeader)
+  }
+}
+
+export const getGraphQLClient = (token?: string) => {
```

Comment:
this should additionally check to see if a token is available and use it if one isn't passed in.

there are no circumstances where we want to avoid the auth if it is available.